Create test_mozilla_org

peiying2 opened the pull request:
Draft file, pending L10n PM review.

peiying2 committed:
Create test_mozilla_org

Draft file, pending L10n PM review.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+
+## Key Links
+* Production: https://www.mozilla.org/(locale)/
+* Staging: https://www-dev.allizom.org/(locale)/
```
Dev server, staging is another url (no need to tell them, is www.allizom.org)

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Production: https://www.mozilla.org/(locale)/
+* Staging: https://www-dev.allizom.org/(locale)/
+* Repository:  https://github.com/mozilla-l10n/www.mozilla.org/{local code}/{project name}/{sub pages}
+* Pontoon: https://pontoon.mozilla.org/(local code)/mozillaorg/{project name}/{sub pages}
```
I would remove the (locale code) from all other URLs, you can point them to https://pontoon.mozilla.org/projects/mozillaorg/ for Pontoon

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Staging: https://www-dev.allizom.org/(locale)/
+* Repository:  https://github.com/mozilla-l10n/www.mozilla.org/{local code}/{project name}/{sub pages}
+* Pontoon: https://pontoon.mozilla.org/(local code)/mozillaorg/{project name}/{sub pages}
+* Locamotion: https://mozilla.locamotion.org/{locale code}/mozilla_lang/{project name}/{sub pages}
```
Same here https://mozilla.locamotion.org/projects/mozilla_lang/

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+# Testing Mozilla.org
+
+Mozilla.org is highly visible because the site houses the basic info of all Mozilla produced products, conveys Mozilla’s mission, vision and values it stands for. Additionally, it promotes initiatives and campaigns in time of these events. As a result, new pages are created. New content is added to existing pages. New layout is introduced. The localized versions reach 60% of the Mozilla users glo ...
```
Later you use the Oxford comma, you should add one before "and values".

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Have the matching US page up as reference, though some strings may not be identical due to A/B testing
+* Have the project you just localized available for editing (Pontoon or Locamotion)
+
+## Linguistic Testing
```
If these titles are supposed to be at the same level of "pre-l10n tests", and below "What to test:", they should be 3rd level.

Other documents don't use Title case for titles, I would keep doing that for consistency.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+
+We highly advise you to ask other community members to conduct peer review not only on Pontoon or Locamotion, but on staging While not all the languages are required for certain projects, each community can opt in the projects at a later time.
+
+## What to test:
```
Drop the colon?

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+## What to test:
+
+### Pre-l10n test
+* Have your [glossary]((https://transvision.mozfr.org/) available as a reference, select mozilla.org as Repository, your language as your Target Locale.
```
Do we really want to call Transvision a glossary?

Inconsistent use of period at the end of the list items.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Check for truncation: button text remains inside the button
+* Header line break wraps at proper place
+* Text not overlapping graphic
+* Terminology is consistency with product, and between web pages
```
consistent

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Locamotion: https://mozilla.locamotion.org/{locale code}/mozilla_lang/{project name}/{sub pages}
+* Web dashboard: https://l10n.mozilla-community.org/webdashboard/{local code}. Visit this page on a regular basis to check localization progress, pending work, deadline, and errors that were introduced during translation. 
+
+We highly advise you to ask other community members to conduct peer review not only on Pontoon or Locamotion, but on staging While not all the languages are required for certain projects, each community can opt in the projects at a later time.
```
Missing period before While

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Brand names remain in English
+* Product names comply to Mozilla guideline and adhere to what the community has agreed to
+* No corrupted characters
+* Click on the links on the page, which should take you to the pages of the same language if they are localized?
```
This is not necessarily true, sometimes pages are in English, also links are never localized so they would be broken for all languages

peiying2 replied:
change it to be more specific: "... or they will be redirected to en-US if the pages are not." If this is still wrong or confusing, we need to change the sentence. The point is that they should check the links.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Product names comply to Mozilla guideline and adhere to what the community has agreed to
+* No corrupted characters
+* Click on the links on the page, which should take you to the pages of the same language if they are localized?
+* Nav bar terms consistent with the page titles they are linked to 
```
There are exceptions, e.g. when the space available in the navigation is smaller

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Nav bar terms consistent with the page titles they are linked to 
+* Footer links don’t overlap with one another
+
+You can make linguistic changes directly in Pontoon, Locamotion or github.  
```
github -> GitHub. 

Where in GitHub? Link the repository somewhere.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+
+## Functionality Testing
+* Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox.
+* Font support and readability
```
Inconsistent period at end of list items.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+## Functionality Testing
+* Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox.
+* Font support and readability
+* Footer: links point to pages of in the same language if those pages, sites or documents are localized
```
See comment above about this not always being the case, also extra "of"

peiying2 replied:
add "If they are not, they should be redirected to en-US pages."

flodolo replied:
Links might external to mozilla.org, on websites that are en-US only. It's a bit hard given how many possibilities there are.

Even if they are on mozilla.org, it might be pages that we don't localize. 

Personally I would drop both sentences, maybe only add a note about testing the links to verify that the translation of the link is coherent and the link is functional.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox.
+* Font support and readability
+* Footer: links point to pages of in the same language if those pages, sites or documents are localized
+* Language list: your language is listed as one of the option
```
Tricky: it's not going to be listed if the page is localized in their own language ;-)

Tell them to visit for example www.mozilla.org/en-US and check

peiying2 replied:
change to "Is your language listed as one of the options? Check www.mozilla.org/en-US to confirm."
I didn't know it is not. should we hook it up with CLDR library?

flodolo replied:
On second thoughts, safer to tell them to check https://www-dev.allizom.org/en-US/

If the home page is not fully localized, their locale won't be listed on prod.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Font support and readability
+* Footer: links point to pages of in the same language if those pages, sites or documents are localized
+* Language list: your language is listed as one of the option
+* Deliberately type a wrong broken link, verify whether 404 page is in the same language.
```
You can provide a link here, instead of letting them guess www-dev.allizom.org/blabla

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Footer: links point to pages of in the same language if those pages, sites or documents are localized
+* Language list: your language is listed as one of the option
+* Deliberately type a wrong broken link, verify whether 404 page is in the same language.
+* Sorting order of languages in dropdown list
```
Uhm, can you explain this?

peiying2 replied:
Remember the bug for contact in the FFOS development days? Same issue. Is the language sorting order by ISO code or in language specific order? This is mostly a problem for non-Latin character based languages, such as CCJK, etc.  I can remove it if the list goes by English alphabet order.

flodolo replied:
What's unclear to me is which dropdown list they should check… Also, it's not necessarily true that a dropdown should be ordered.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Language list: your language is listed as one of the option
+* Deliberately type a wrong broken link, verify whether 404 page is in the same language.
+* Sorting order of languages in dropdown list
+* In RTL (right to left languages), both target language and English flows in the correct directions.  The mirroring effect is correct as defined by Mozilla UX team.  Indentation and alignment are correct.
```
Not sure I understand the part about UX team, localizers for RTL languages know how the page should look.

Nit: keep using one single space after periods.

peiying2 replied:
From what I understand, the "mirror" effect is not exactly 100% flip over horizontally.  I am not clear if we follow industry standard or Mozilla standard, this is why i made it vague, as the standard is not unified (I can't recall all the details", but as long as the localizers, especially new comers know what it is, it is fine.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* In RTL (right to left languages), both target language and English flows in the correct directions.  The mirroring effect is correct as defined by Mozilla UX team.  Indentation and alignment are correct.
+
+
+## Testing for Fundraising Campaigns: 
```
Should we include this? It seems overly specific

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+* Payment system available for the targeted country
+* Country list in your language and order
+
+## Compatibility testing:
```
Is this really on localizers to test? It might be, especially for languages not using Latin characters, but it's border line.

peiying2 replied:
we can include Theo to review this. It is once a year, but the list has expanded. There is a mixture of things that are not implemented fully.  Especially payment, community can help. If this is just too much, we can remove it.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+
+## Compatibility testing:
+* Test the page layout in other major browsers and on other platform
+* Test the page layout on the leading home grown browser if available.
```
home grown browser? 🐖

peiying2 replied:
You know what I meant, lol!!  Are there major browsers that the Russians, the Brazilians, and the Middle Easterners that developed that are very popular locally/regionally? Let's rephrase it!!

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,67 @@
+
+## When Can I See the Localized Page on Production? 
+
+Code push to production is made almost daily.  When a brand new page is being localized, the page won't go live until it is fully localized for the very first time. Once a page is on the production, an ongoing maintenance is required, as updates take place on a irregular basis. When it is updated with minor change, especially when a new string is associated with a tag, the page will be updated wh ...
```
the production -> production, or the production server
webpage dashboard -> web dashboard

I don't think we should explain tags to them, they don't see them. We should just tell them that new strings added to existing pages are visible in production only when localized, unless they're in a shared file.

For the last part, the concept is that sometimes a completely new template is used for a page, e.g. the home page, and that deadlines are strict: after the deadline, the old template will be disabled. All these big changes are communicated well in advance on the mailing list.

peiying2 committed:
Fixes and updates per comments

- Fix the punctuation
- Update the links 
- Rephrase certain items to make them clearer, hopefully

peiying2 committed:
Update testing_mozilla_org.md

- Revision on the "footer link" item in functional testing
- Language availability in dropdown menu (check on dev server instead of prod server)
- Remove mentioning of sorting order of language dropdown list.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+# Testing Mozilla.org
+
+Mozilla.org is highly visible because the site houses the basic info of all Mozilla produced products, conveys Mozilla’s mission, vision, and values it stands for. Additionally, it promotes initiatives and campaigns in time of these events. As a result, new pages are created. New content is added to existing pages. New layout is introduced. The localized versions reach 60% of the Mozilla users gl ...
```
produced product -> products

I would drop the part `As a result, new pages are created. New content is added to existing pages. New layout is introduced.` because it sounds a bit weird in this kind of document, but that's just me.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+Mozilla.org is highly visible because the site houses the basic info of all Mozilla produced products, conveys Mozilla’s mission, vision, and values it stands for. Additionally, it promotes initiatives and campaigns in time of these events. As a result, new pages are created. New content is added to existing pages. New layout is introduced. The localized versions reach 60% of the Mozilla users gl ...
+
+## Key Links
+* Production: https://www.mozilla.org/(locale)/
```
Can you drop (locale) from this link? If you look at the page you'll understand why
https://github.com/peiying2/documentation/blob/89533e7b293a78cec9ff0a807bc92c4b6939db10/products/mozilla_org/test_mozilla_org.md

Keep a working URL here, add a note later that pages on mozilla.org should redirect to the right locale, if they don't you can add your locale code. Same for the webdashboard link later.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+
+## Key Links
+* Production: https://www.mozilla.org/(locale)/
+* Staging: https://www-dev.allizom.org/(locale)/
```
Staging -> Dev server

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+## Key Links
+* Production: https://www.mozilla.org/(locale)/
+* Staging: https://www-dev.allizom.org/(locale)/
+* Repository:  https://github.com/mozilla-l10n/www.mozilla.org/
```
Double space. You also need to explain what this repository is, and why the shouldn't bother if they work on tools.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+
+### Pre-L10n Test
+* Have your [glossary]((https://transvision.mozfr.org/) available as a reference, select mozilla.org as Repository, your language as your Target Locale
+* Associate the page with a product or site if it is localized. (e.g.: Firefox, Test Pilot)
```
Remove period after localized

TheoChevalier replied:
Not sure I understand this sentence. By associate, we mean “Make the content of the page consistent with product translations”?

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+### Pre-L10n Test
+* Have your [glossary]((https://transvision.mozfr.org/) available as a reference, select mozilla.org as Repository, your language as your Target Locale
+* Associate the page with a product or site if it is localized. (e.g.: Firefox, Test Pilot)
+* Have the matching US page up as reference, though some strings may not be identical due to A/B testing
```
US -> English

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+
+### Linguistic Testing
+* Translation quality in context
+* Grammatically correct in context
```
This one is missing a subject (correct -> correctness)

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+* Check for truncation: button text remains inside the button
+* Header line break wraps at proper place
+* Text not overlapping graphic
+* Terminology is consistent with product, and between web pages
```
among

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+* Nav bar terms consistent with the page titles they are linked to (except when Nav bar term is shortened due to space limitation)
+* Footer links don’t overlap with one another
+
+You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.org/projects/mozillaorg/), [Locamotion](https://mozilla.locamotion.org/projects/mozilla_lang/) or [GitHub](https://github.com/mozilla-l10n/www.mozilla.org/).  
```
comma before or

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+* Font support and readability
+* Footer: verify that the translation of the link is coherent and the link is functional.
+* Language list: Is your language listed as one of the options? Check https://www-dev.allizom.org/en-US/ to confirm.
+* Deliberately type a wrong broken link, verify whether [404 page](https://www-dev.allizom.org/404/) is in the same language
```
As said in the previous comment, just give an example of broken URL instead of asking them to come up with one.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+### Functionality Testing
+* Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox
+* Font support and readability
+* Footer: verify that the translation of the link is coherent and the link is functional.
```
Consistency: final period or not?

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+* Footer: verify that the translation of the link is coherent and the link is functional.
+* Language list: Is your language listed as one of the options? Check https://www-dev.allizom.org/en-US/ to confirm.
+* Deliberately type a wrong broken link, verify whether [404 page](https://www-dev.allizom.org/404/) is in the same language
+* In RTL (right to left languages), both target language and English flows in the correct directions.  The mirroring effect is correct as defined by Mozilla UX team. Indentation and alignment are correct
```
No double spaces after period, one empty line after is enough.

Again, localizers are not supposed to know that the UX team wants or decides.

```
If your language is RTL, make sure that the page layout and text flow in the correct direction.
```

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+* Payment system available for the targeted country
+* Country list in your language and order
+
+### Compatibility Testing:
```
I would really limit this to testing that mozilla.org look good in other major browser if your language doesn't use Latin characters.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,66 @@
+
+## When Can I See the Localized Page on the Production Server? 
+
+Code push to the production server is made almost daily.  When a brand new page is being localized, the page won't go live until it is fully localized for the very first time. Once a page is on the production server, an ongoing maintenance is required, as updates take place on a irregular basis. New strings added to existing pages are visible in the production server only when localized, unless t ...
```
We need to simplify this, e.g.

>Updated translations are pushed to the production server almost daily. 
>When a brand new page is available for localization, it won't be enabled in production until it's fully localized. When existing pages receive updates with new strings, this new content won't be displayed on production until localized, to avoid displaying a mix of English and localized text. 
In some cases pages receive major updates that require a complete rewrite of the template: if this happens, the old template is kept online only for a short period of time and, when removed, it will cause the URL to redirect users to the English version.

flodolo reviewed (changes requested):
In general, look out for consistency:
* Is there an empty line after titles or not? A period after list items?
* Don't use title case and colon in titles, nor double spaces after a period.

peiying2 committed:
Update #3 of test_mozilla_org.md

Add the syncs frequencies between repo, l10n tools and server.

TheoChevalier commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,73 @@
+* Postal/zip code in the proper format
+* Mailing address format appropriate for the country
+* Payment system available for the targeted country
+* Country list in your language and order
```
We do not display a country list on the take over, nor we do on donate.mozilla.org

peiying2 replied:
The country list is definitely on the form when people fill out personal info. but if this is provided by third party, then we don't have any control. Having said that, the donation process expects donors to understand English ^-^. People will not provide personal info if the process is not in their language, especially with money involved.

TheoChevalier commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,73 @@
+* Currency availability
+* Postal/zip code in the proper format
+* Mailing address format appropriate for the country
+* Payment system available for the targeted country
```
Duplicate with “Currency availability”, we automatically enable Stripe and/or Paypal based on the currency availability (see links above)

TheoChevalier commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,73 @@
+The annual fundraise campaign is localized in many languages, but not all languages. 
+* Date and time format is country specific
+* Currency availability
+* Postal/zip code in the proper format
```
We don’t have this form on mozilla.org

peiying2 replied:
I will send you a screenshot on IRC. If this is something created by the 3rd party, then it is out of our control.

TheoChevalier commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,73 @@
+* Date and time format is country specific
+* Currency availability
+* Postal/zip code in the proper format
+* Mailing address format appropriate for the country
```
We don’t have this form on mozilla.org either, it is directly handled by the implementation on donate.mozilla.org with Stripe and Paypal (and we don’t have control over those forms).

peiying2 replied:
if this is the case, then we can remove it.

TheoChevalier commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,73 @@
+
+### Testing for Fundraising Campaigns: 
+The annual fundraise campaign is localized in many languages, but not all languages. 
+* Date and time format is country specific
```
Is “Give before December 31st” the only date displayed on mozilla.org?

peiying2 replied:
I think on the confirmation receipt of the transaction.  However, I only went as far as the mailing address window in the flow.

TheoChevalier commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,73 @@
+
+
+### Testing for Fundraising Campaigns: 
+The annual fundraise campaign is localized in many languages, but not all languages. 
```
Yes, but it is open to all locales (both donate.mozilla.org and the strings on mozilla.org takeover). We can add more currencies as we add locales, but it is true that we are limited by the  currencies supported by Stripe and Paypal. We can support any of those for Stripe: https://support.stripe.com/questions/which-currencies-does-stripe-support#currencygroup1
And any of those for Paypal: https://developer.paypal.com/docs/classic/api/currency_codes/#paypal

TheoChevalier replied:
Just to clarify the whole section, this is the only thing hosted on mozilla.org: https://drive.google.com/file/d/0B20-0lpP23c0NUlnQW1BSVhGOHM/view

Everything that comes when you click on Donate is from donate.mozilla.org and does not belong to this doc ;)

TheoChevalier commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,73 @@
+* In RTL (right to left languages), both target language and English flows in the correct directions.  The mirroring effect is correct as defined by Mozilla UX team. Indentation and alignment are correct
+
+
+### Testing for Fundraising Campaigns: 
```
We should also mention that those strings are not always available for testing, and that the takeover only appears once in a given tab.

peiying2 replied:
I honestly don't know the flow of this part of the testing. I know it involves transaction.  From the input below, the solution is provided by a 3rd party.  Maybe we can discuss this on our weekly call so I understand it better.

TheoChevalier commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,73 @@
+* Language list: Is your language listed as one of the options? Check https://www-dev.allizom.org/en-US/ to confirm.
+* Deliberately type a wrong broken link, verify whether [404 page](https://www-dev.allizom.org/404/) is in the same language
+* In RTL (right to left languages), both target language and English flows in the correct directions.  The mirroring effect is correct as defined by Mozilla UX team. Indentation and alignment are correct
+
```
Should there be a new section here to explain how to test strings difficult to display? Like strings in firefox_tour.lang or privacy_tour.lang for either tracking protection tour, whatsnew, firstrun…

peiying2 replied:
We can call out certain type of strings that are difficult to test.   Do you have a list of these pages in mind? What makes them difficult to test?

TheoChevalier commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -0,0 +1,73 @@
+* Repository:  https://github.com/mozilla-l10n/www.mozilla.org/
+* Pontoon: https://pontoon.mozilla.org/projects/mozillaorg/
+* Locamotion: https://mozilla.locamotion.org/projects/mozilla_lang/
+* Web dashboard: https://l10n.mozilla-community.org/webdashboard/{local code}. Visit this page on a regular basis to check localization progress, pending work, deadline, and errors that were introduced during translation. 
```
Wrong link + typo, should be `https://l10n.mozilla-community.org/webdashboard/?locale={locale code}`

peiying2 committed:
Update #4 test_mozilla_org.md

- Incorporate text update recommended by Flod
- Revised ambiguity in sections mentioned by Theo and Flod
- Corrected some URLs
- Add sync and update frequency section
- Removed Fundraising campaign section all together after discussion with Theo.  It belongs to a different doc with a different test focus
- Will add testing instruction for certain pages or features that are not static.  Identify the pages and add the test steps incrementally to this doc

peiying2 committed:
Update #5 testing_mozilla_org.md 

Hopefully I caught everything.
